docs/tools/lsp-smart-contracts/constants.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 4
+---
+
+# Constants
+
+The [`constants.js`](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/constants.js) file exposes many useful variables.
```

Comment:
Maybe we can add a code snippet here like in the README to show the consstants we can import?

```js
import {
    INTERFACE_IDS,
    ERC1271,
    OPERATIONS,
    SupportedStandards,
    ERC725YKeys,
    BasicUPSetup_Schema,
    PERMISSIONS,
    ALL_PERMISSIONS,
    Errors,
    EventSignatures,
} from "@lukso/lsp-smart-contracts/constants.js";
```